src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/OnlineHearingSteps.java

```diff
@@ -310,6 +313,63 @@ public void theOnlineHearingReasonIsReason(String reason) throws Throwable {
         assertEquals(reason, getOnlineHearingResponse().getRelistReason());
     }
 
+    @When("^(drafting|issuing) the relist$")
```

Comment:
Wouldn't it be easier if this simply said
the relist state is '<ACTUAL STATE THAT WOULD BE SET>'?

Reply:
There are different styles of writing BDD scenarios:

A:
```gherkin
Given the relist state is 'drafted'
And the relist reason is 'Whatever'
When a PUT request is sent to online hearing relist
```

B:
```gherkin
When drafting a relist with reason "Whatever"
```

They would probably do the same thing underneath, however, the second one communicates intent without overloading the reader with too much technical details. It is easier to comprehend.